ods_ci/tests/Resources/OCP.resource

```diff
@@ -97,3 +97,8 @@ Fetch CA Certificate If RHODS Is Self-Managed
     IF  ${self_managed}==${TRUE}
         Fetch Openshift CA Bundle
     END
+
+Wait For Pods To Be Ready
+    [Arguments]    ${label_selector}    ${namespace}    ${timeout}=300s
+    ${rc}    ${out}=    Run And Return Rc And Output
+    ...    oc wait --for=condition=ready pod -l ${label_selector} -n ${namespace} --timeout=${timeout}
```

Comment:
It will be better if you add check for the output in case timeout is exhausted

Reply:
in general, i didn't mean to block execution untilt the install code gets stability, since there might be some changes in near future. I would add the blocking check in critical part like this one

Reply:
in fact, in another run after the blocking check was added, the test was stopped because a delay in getting the pods started. 

The waiting should be improve to wait for the pod first of all. We actually have another custom keyword `Wait Until Operator Pods Are Running` which should do the job. The problem with that one is that it requires a lot of input (list of all the expected pods and containers etc) which are a lot in this scope...Wdyt?

Reply:
Hmm ..I would suggest to check the rc outpu with run keyword and continue on failure keyword until we get the stability

Reply:
installation is now more stable, I extended the kw to check pods